demos/basic.demo

```diff
@@ -0,0 +1,34 @@
+# Demo: how to install, configure and use Knvavigator in a local k8s cluster, such as minikube or KinD
+
+# Show the cluster infomation
+kubectl cluster-info
+
+kubectl get nodes
+
+# Intall KWOK node simulator 
+
+KWOK_REPO=kubernetes-sigs/kwok
+KWOK_LATEST_RELEASE="v0.5.2"
+kubectl apply -f "https://github.com/${KWOK_REPO}/releases/download/${KWOK_LATEST_RELEASE}/kwok.yaml"
+kubectl apply -f "https://github.com/${KWOK_REPO}/releases/download/${KWOK_LATEST_RELEASE}/stage-fast.yaml"
+kubectl apply -f https://github.com/${KWOK_REPO}/raw/main/kustomize/stage/pod/chaos/pod-init-container-running-failed.yaml
+kubectl apply -f https://github.com/${KWOK_REPO}/raw/main/kustomize/stage/pod/chaos/pod-container-running-failed.yaml
+
+# Add virtual nodes to the cluster
+helm install virtual-nodes charts/virtual-nodes -f charts/virtual-nodes/values.yaml
```

Comment:
Since values.yaml is now empty, do you want to create values.yaml in the demo directory? That was you can control the node configuration for the demo.